src/components/ContentOpenWith/__tests__/IntegrationPortal-test.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import IntegrationPortal from '../IntegrationPortal';
+
+const INTEGRATION_CONTAINER_ID = 'integration-container';
+
+describe('components/ContentOpenWith/IntegrationPortal', () => {
+    const getWrapper = props => shallow(<IntegrationPortal {...props} />);
+
+    it('should use an existing container if possible', () => {
+        const container = document.createElement('div');
+        container.id = INTEGRATION_CONTAINER_ID;
+        document.body.appendChild(container);
+
+        const wrapper = getWrapper({
+            integrationWindow: {
+                // use normal document in our integrationWindow
+                document,
+            },
+        });
+
+        const instance = wrapper.instance();
+        expect(instance.containerElement).toEqual(container);
+    });
+
+    it('should create the container if needed, and assign an id ', () => {
+        const wrapper = getWrapper({
+            integrationWindow: {
+                // use normal document in our integrationWindow
+                document,
+            },
+        });
+
+        wrapper.instance();
```

Comment:
The Enzyme docs mention this is used to return the underlying instance. Does calling it in isolation have a desirable side effect? Should we use `mount`, instead?

Reply:
you're correct, that call isn't needed in this case.